feature-libs/product/configurators/rulebased/cpq/cpq-configurator-serializer.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { Type } from '@angular/core';
+import { TestBed } from '@angular/core/testing';
+import { CpqConfiguratorSerializer } from './cpq-configurator-serializer';
+//import { Cpq } from './cpq.models';
+//import { Configurator } from './../core/model/configurator.model';
+
+describe('CpqConfiguratorSerializer', () => {
```

Comment:
There are no tests for **cpq-configurator-serialize**

Reply:
@AlexanderKuschner: please provide a test

Reply:
done